utils/build_swift/resources/SwiftPM-Unified-Build.xcworkspace/contents.xcworkspacedata

```diff
@@ -1,19 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Workspace
    version = "1.0">
-   <FileRef
-      location = "group:../../../../swift-syntax">
-   </FileRef>
-   <FileRef
-      location = "group:../../../../swift-stress-tester/SourceKitStressTester">
-   </FileRef>
-   <FileRef
-      location = "group:../../../../swift-format">
-   </FileRef>
-   <FileRef
-      location = "group:../../../../swift-stress-tester/SwiftEvolve">
-   </FileRef>
-   <FileRef
-      location = "group:../../../../swift-docc">
-   </FileRef>
+   <FileRef location = "group:../../../../indexstore-db"></FileRef>
+   <FileRef location = "group:../../../../llbuild"></FileRef>
+   <FileRef location = "group:../../../../sourcekit-lsp"></FileRef>
+   <FileRef location = "group:../../../../swift-argument-parser"></FileRef>
+   <FileRef location = "group:../../../../swift-collections"></FileRef>
+   <FileRef location = "group:../../../../swift-docc"></FileRef>
+   <FileRef location = "group:../../../../swift-driver"></FileRef>
+   <FileRef location = "group:../../../../swift-format"></FileRef>
+   <FileRef location = "group:../../../../swift-stress-tester/SourceKitStressTester"></FileRef>
+   <FileRef location = "group:../../../../swift-stress-tester/SwiftEvolve"></FileRef>
+   <FileRef location = "group:../../../../swift-syntax"></FileRef>
+   <FileRef location = "group:../../../../swift-system"></FileRef>
+   <FileRef location = "group:../../../../swift-tools-support-core"></FileRef>
+   <FileRef location = "group:../../../../swiftpm"></FileRef>
+   <FileRef location = "group:../../../../yams"></FileRef>
```

Comment:
Should this just be adding indexstore/lsp?

Reply:
sourcekit-lsp brings in additional dependencies that are also added here
* swift-argument-parser
* swift-tools-support-core
* swiftpm
  * llbuild
  * swift-system
  * swift-driver
    * yams
  * swift-collections

Actually, are we missing swift-crypto @ahoppen? https://github.com/apple/swift-package-manager/blob/73133d3f69d956f16adde38c07c6969e02d6e5a1/Package.swift#L606

Reply:
I thought I had walked the entire dependency tree but apparently I missed that. Thanks for noticing

Reply:
Oh, of course. I was only thinking about indexstore-db for some reason (I tagged on lsp at the end without really thinking about it). Thanks!